src/thruster_types/thruster_types.proto

```diff
@@ -1,6 +1,19 @@
 package rock_thruster.msgs;
 
-message ThrusterInput
+message Raw
 {
-    optional double     frequency = 1 [default = 0];
+    optional string         name = 1;
```

Comment:
When I use two separate types it is easy to "decode" the message.
If I create only the Thruster msg I think I would need a "mode" attribute (am I wrong?)

Thruster
{
   optional string name;
   optional string mode;
   optional double raw;
   optional double effort;
}